activitypub.php

```diff
@@ -17,12 +17,13 @@
 
 use function Activitypub\site_supports_blocks;
 
-\defined( 'ACTIVITYPUB_REST_NAMESPACE' ) || \define( 'ACTIVITYPUB_REST_NAMESPACE', 'activitypub/1.0' );
```

Comment:
You initially put that outside of the loader, to prevent activation errors: https://github.com/Automattic/wordpress-activitypub/commit/6a0fc43a05e56308965854126be745d17aaf109e

Is this no longer an issue?